shared/src/main/scala/org/atnos/eff/FutureEffect.scala

```diff
@@ -24,6 +24,15 @@ trait FutureCreation {
 
   def async[R, A](a: =>A)(implicit m: Member[Future, R], ec: ExecutionContext): Eff[R, A] =
     send(Future(a))
+
+  def lift[R, A](
```

Comment:
We'd better call this `liftFuture` to avoid having import conflicts if we adopt a similar technique with another effect and want to import `org.atnos.eff.all._`.